.github/workflows/ci.yml

```diff
@@ -1,18 +1,17 @@
 name: CI
-
 on:
   push:
     branches: ["master"]
   pull_request:
     branches: ["master"]
 
 jobs:
-  build:
+  build_agent_controller:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x]
```

Comment:
Curious why node 14 is no longer supported?

Reply:
wasnt building in CI/CD due to my  machine creating a newer version of lockfile?